Dockerfile

```diff
@@ -21,7 +21,7 @@ COPY ./Composer .
 COPY ./extensions ../extensions
 # run yarn install as a distinct layer
 RUN yarn install --frozen-lock-file $YARN_ARGS
-ENV NODE_OPTIONS "--max-old-space-size=4096"
+ENV NODE_OPTIONS "--max-old-space-size=10240"
```

Comment:
Do we need this?

Reply:
This is what was making the "Docker Build verification" step keep failing in the automated steps. I don't know if 10240 is the right number, but we can decrease it later if we need to.